crates/weaver_forge/src/config.rs

```diff
@@ -298,6 +301,49 @@ impl Default for TemplateSyntax {
     }
 }
 
+/// Whitespace control configuration for the template engine.
+#[derive(Deserialize, Debug, Clone)]
+pub struct WhitespaceControl {
+    /// Configures the behavior of the first newline after a block.
+    /// See <https://docs.rs/minijinja/latest/minijinja/struct.Environment.html#method.set_trim_blocks>
+    #[serde(default = "default_trim_blocks")]
```

Comment:
I thought if we want default to be false/0/empty string etc we don't need to define anything as it's default behaviour